recipes-devtools/deepstream/deepstream-6.1_6.1.0-1.bb

```diff
@@ -82,6 +89,32 @@ do_install() {
     cp -R --preserve=mode,timestamps ${S}${DEEPSTREAM_PATH}/sources/includes/* ${D}${includedir}/
 
     cp -R --preserve=mode,timestamps ${S}${DEEPSTREAM_PATH}/sources/ ${D}${DEEPSTREAM_PATH}/
+
+    # XXX---
+    # Some of the binaries are not using the right SONAME
+    # in its DT_NEEDED ELF header, so we have to rewrite it to prevent
+    # a broken runtime dependency.
```

Comment:
This is a bit concerning, as substituting a different version of a library (particularly C++) could result in strange runtime errors.

Looks like most of these are for the `libyaml-cpp` version difference.  Maybe we could instead add a recipe for the older version (that could co-reside with the upstream version, not replace) and add that as a dependency?  We've done that for other cases like this.

Reply:
Yes, I can do that 
I'm doing some tests and I will share here